pyproject.toml

```diff
@@ -0,0 +1,101 @@
+[tool.poetry]
+name = "thunderbird"
+version = "1.2.3"
+description = "A Web Processing Service for Climate Explorer data preparation"
+authors = [
+    "Nikola Rados",
+    "Sangwon Lim",
+    "Cairo Sanders",
+    "James Hiebert <hiebert@uvic.ca",
+    "Eric Yvorchuk <eyvorchuk@uvic.ca",
+    "Long Vu",
+]
+license = "GPL3"
+readme = "README.md"
+homepage = "http://www.pacificclimate.org/"
+repository = "https://github.com/pacificclimate/thunderbird"
+keywords = ["wps", "pywps", "birdhouse", "thunderbird"]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX",
+    "Programming Language :: Python",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Scientific/Engineering :: Atmospheric Science",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+]
+packages = [
+    {include = "thunderbird"},
+]
+include = [
+    "tests/data/*.nc",
+    "tests/metadata-conversion/*.yaml",
+]
+
+[[tool.poetry.source]]
+name = "pcic"
+url = "https://pypi.pacificclimate.org/simple"
+priority = "supplemental"
+
+[tool.poetry.dependencies]
```

Comment:
I see some exact version specs here (e.g., `cftime = "1.6.3"`), which are not desirable unless they are necessary to prevent installation errors. Please check whether you can loosen them.

It looks as if these are all or mostly top-level deps, yay. However, if there are any subsidiary deps, please see if you can remove them. Minimal dependencies are a good goal.